dynawo/sources/Modeler/DataInterface/PowSyblIIDM/CPPLINT.cfg

```diff
@@ -1 +1,2 @@
 filter=-build/c++11
+filter=-runtime/string
```

Comment:
What is this rule?

Reply:
because of this rule, I cannot have a static const string member.

Reply:
can we avoid this static const string?

Reply:
No

Reply:
Are we talking about IIDMExtensionTrait::name ? What about using a const char* as cpplint recommend?

Reply:
OK for some reasons, now it works using const char[] instead of string so I rollback that